lips/lip-6.mdx

```diff
@@ -101,35 +105,35 @@ The following transaction scripts can be run by the Validator Owner:
+For a Validator Owner who is not yet added to the Validator Set, the Validator Operator (or Validator Owner if `operator_account` is unset) may change the `config` by calling a transaction script `register_validator_config`. This script takes an address of the Validator Owner (to avoid confusion when one Validator Operator runs Validator Nodes for multiple Validator Owners), a new consensus public key (a 32 elements byte array) and two byte arrays for network addresses: `validator_network_addresses` for other Validator Operators to communicate with this Validator Operator during consensus and `fullnode_network_addresses` for Full Nodes, clients or validators who are catching-up.
 
-If a Validator Owner is in the Validator Set, the `operator_account` may update the `config` both locally and in the Validator Set by sending `set_validator_config_and_reconfigure` transaction script, this transaction triggers the Reconfiguration.
+If a Validator Owner is in the Validator Set, the `operator_account` may update the `config` both locally and in the Validator Set by sending `set_validator_config_and_reconfigure` transaction script, this transaction script triggers the Reconfiguration.
 
 ## Validator Owner to be added/removed from the Validator Set
-Once the Validator Owner or its Validator Operator sets the `config` of the Validator Node and is ready to participate in the consensus protocol, it should ask the AOS off-chain to be added to the Validator Set. The AOS sends a transaction from the LibraRoot account: `add_validator_and_reconfigure`, once it executes, it triggers the Reconfiguration, other validators connect to the Validator Node and it starts participating in the protocol. Note that the size of the Validator Set and hence the size of the quorum for consensus decisions increases.
+Once the Validator Owner or its Validator Operator sets the `config` of the Validator Node and is ready to participate in the consensus protocol, it should ask the AOS off-chain to be added to the Validator Set. The AOS sends a transaction from the LibraRoot account: `add_validator_and_reconfigure`. Once this transaction executes, it triggers the Reconfiguration, other Validator Operators connect to the Validator Node and the Validator Opeerator starts participating in the protocol. Note that the size of the Validator Set and hence the size of the quorum for consensus decisions increases.
```

Comment:
A Validator Node is consensus-ready once the Validator Operator sets the `config` and the node is operational. At which point, AOS can submit a transaction from the LibraRoot account: ...